driver/result_set.cpp

```diff
@@ -131,7 +159,7 @@ std::size_t ResultSet::fetchRowSet(SQLSMALLINT orientation, SQLLEN offset, std::
     }
 
     if (prefetched_rows.size() < size) {
-        constexpr std::size_t prefetch_at_least = 10'000;
+        constexpr std::size_t prefetch_at_least = 100;
```

Comment:
Could you please explain why 100 ? and Why you've changed that ?

Reply:
It was 100 historically. Initially I raised it, just empirically. Initially this had an effect on the initial display size, as it was calculated dynamically always, so the number of rows you prefetch would cause the first reported display size "quality". Currently, rolling back to what was before, partially because neither value have any significant effect on performance (this is another story - the strategy of stream reading have to be completely reviewed and optimized), however the effect of this on display size is fully removed by this PR.

Reply:
Ok